setup.py

```diff
@@ -10,6 +10,7 @@
     zip_safe=False,
     install_requires=[
         'boto3',
+        'cffi>=1.15.1',
```

Comment:
Forcing this to be pinned to the current latest package version to resolve issues with building on aarch64.

Reply:
This change is no longer present as I just used `pip-compile` to upgrade the package without making a change to `setup.py`.